### PR TITLE
Create reality-shock-video-no-1.md

### PR DESCRIPTION
**Pull request by georgeshk**
Concept and rough script of the first Reality Shock video, dedicated to smartphone addiction.

### CODE REVIEW

**Comment by aschrijver** [comment]
Cool, @georgeshk this looks great

Couple of general improvements / considerations:

- Let's consider each video a separate campaign
- Let's give each campaign as short name / working title
  - Each campaign can then go to a separate subfolder of `campaigns` named `working-title`
  - The root document in the subdir can be named `README.md` (so it is displayed by default)

Can you make those changes, or do you prefer I do it?
First we need a title for the campaign, let's discuss this in the issue you created.

File: campaigns/reality-shock-video-no-1.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Reality Shock - Video #1 - Concept and Script
+
+## Reminder: Campaign theme
+
+The overarching theme of the campaign is to provoke and make people ponder the question: "Are you really comfortable with this?"
+
+## Concept
+
+“Let us look at how our pastimes have changed by considering a scene at a cafe that represents two decades. From leasurely and seemingly tranquil activities, we have become increasingly absorbed in our phones, in the process building the wealthiest and most powerful companies in the world’s history. At the end, we realize our phone has become our only source of gratification.”
```
Comment (aschrijver) [inline campaigns/reality-shock-video-no-1.md]: leasurely = leisurely

File: campaigns/reality-shock-video-no-1.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
...fee place terraces you can see everywhere in Europe. This kind of place used to be cliche for spending time leasurely with friends, reading newspapers or a book, and watching people. The music starts slowly. Maybe only a few patrons (2 or 3, importantly most of them in their early twenties - see note below), all of them engaging in above-mentioned, "traditional" activities. Then as the music gathers pace and grows crescendo, more and more patrons of all ages and all backgrounds (students, business people, etc), alone of in groups, sit in, some of them picking up their phones and getting busy. The music grows louder, more and more patrons arrive, this time already holding their phones as they arrive. We start superimposing data (in the form of these high-tech transparent windows) pertaining to each patron that is relevant for harvesting, and showing growing profits, market caps for tech companies, etc. Once the music finally calms down in the very few seconds (just before 2:37), one person who was engrossed with his/her phone looks up from his/her screen, and realizes there is not a soul around. We can conclude with showing a message/question for the viewer on these few final notes.
```
Comment (aschrijver) [inline campaigns/reality-shock-video-no-1.md]: typo: alone _or_ in groups

**Comment by georgeshk** [comment]
Hi Arnold,

Not quite sure what you have in mind, and a bit tricky for me to take care of as I cant seem to be able to create folders or create files without using a branch and pull request.

I was thinking all short videos to be one campaign under same theme, which you called "Reality shock". I kind of like the name and adopted it.

Maybe under "Campaigns", we could have "Reality Shock" and "Phone Zombies". Then under "Reality Shock", we would have "Video 1 - Mankind 2.0", "Video 2 - Data breach", "Video 3 - Social credits", etc.

All video subfolders would contain all files. I am not in favor of mutliple levels of arborescence.

Under "Strategy", we would have "Production general issues", "Crowdfunding general issues", "Distribution general issues", with all project-specific videos in their respective folders (i.e. under "Campaigns")

Hope you will find this ok.

Cheers,
Georges




Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by georgeshk** [comment]
Hi Arnold,

Can you make me a collaborator, which would allow to create folders and write content without forking put and requesting a pull?

Cheers,
Georges

**Pull request closed without merging**